adapter-excel/build.gradle

```diff
@@ -1,27 +1,45 @@
+buildscript {
+    ext {
+        springBootVersion = '2.0.3.RELEASE'
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+    }
+}
+
 group 'ch.prevo.open'
 version '0.0.1-SNAPSHOT'
 
 apply plugin: 'java'
+apply plugin: 'io.spring.dependency-management'
+apply plugin: 'org.springframework.boot'
 
 sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
 
-sourceSets {
-    test {
-        resources {
-            exclude '**/~$**'
-        }
-    }
-}
-
 dependencies {
+    compile(project(":node"))
```

Comment:
can we change this to a runtime dependency? Build/Tests work without it

Reply:
It is temporary anyway, because with SPI it will look different. Leave it for now.